scripts/qa_crowbarsetup.sh

```diff
@@ -1525,7 +1525,7 @@ EOF
         /opt/dell/bin/json-edit -a attributes.network.networks.public.ranges.host.end -v 10.162.211.191 $netfile
     fi
     # Setup network attributes for custom MTU
-    if [[ $want_mtu_size -ne 1500 ]]; then
+    if [[ $want_mtu_size ]] && [[ $want_mtu_size -ne 1500 ]]; then
```

Comment:
btw. this is duplicating the default value, so requires changes at two places when the default changes - should be prevented